.travis.yml

```diff
@@ -0,0 +1,34 @@
+sudo: required
+
+language: python
+
+services:
+  - docker
+
+env:
+  global:
+    - PORTS="8069:8069"
+    - LINKS="db:db"
+    - DAEMONIZE="1"
+    - ODOO_VERSION="10.0"
```

Comment:
@Yajo - I'm not sure of your Dockerhub deploy strategy, but this can handle that too. You'll need to do that if you want though - Take a look [at the sample Travis](https://github.com/LasLabs/docker-quality-tools/blob/master/sample_files/.travis.yml#L9). Variables to note:
* DOCKER_REPO
* DOCKER_USER
* DOCKER_TAG
* $DOCKER_PASS_TOKEN
* HUB

[An example of it in use](https://github.com/LasLabs/docker-alpine-jira/blob/master/.travis.yml#L20)